app/scripts/services/projects.js

```diff
@@ -23,6 +23,7 @@ angular.module('openshiftConsole')
           return  AuthService
                     .withUser()
                     .then(function() {
+                      AuthorizationService.getProjectRules(projectName);
```

Comment:
If the place we use it in the views is given a promise when the rules haven't resolved yet, does that not trigger a digest loop?